diskmaker/controllers/lv/reconcile.go

```diff
@@ -49,7 +49,7 @@ import (
 // It also ensures that only stable device names are used.
 
 var (
-	checkDuration = 5 * time.Second
+	checkDuration = 60 * time.Second
 	diskByIDPath  = "/dev/disk/by-id/*"
```

Comment:
Did we had chance to measure impact of the change on CPU usage of diskmaker pods btw?

Reply:
Roughly speaking I would expect some dip in attached chart which is available in OCP dashboard. 
![Screenshot from 2021-09-01 21-47-31](https://user-images.githubusercontent.com/278/131768300-bdfef7b0-a1ed-491f-9e51-51ae3de99eb9.png)